src/test-gui/java/de/qabel/desktop/ui/about/AboutPage.java

```diff
@@ -0,0 +1,33 @@
+package de.qabel.desktop.ui.about;
+
+import com.sun.javafx.robot.FXRobot;
+import de.qabel.desktop.ui.AbstractPage;
+import org.testfx.api.FxRobot;
+
+import static org.junit.Assert.assertTrue;
+
+public class AboutPage extends AbstractPage {
+    private AboutController controller;
+
+    public AboutPage(FXRobot baseFXRobot, FxRobot robot, AboutController controller) {
+        super(baseFXRobot, robot);
+        this.controller = controller;
+    }
+
+    public void btnShowPopup() {
+        clickOn("#btnThanks");
+        setPopupText();
+        showAboutPopup();
+        waitUntil(() -> controller.popupController.textAreaPopup != null);
+        assertTrue(controller.popupController.aboutPopup.isVisible());
+    }
+
+    private void setPopupText() {
+        controller.popupController.setTextAreaContent("test thanks file");
```

Comment:
you never assert that text. shouldn't the controller set a text? why is the test doing this?